src/box/memtx_tree.cc

```diff
@@ -2417,14 +2419,13 @@ tree_read_view_iterator_position_func(struct index_read_view_iterator *it,
 						pos, size);
 }
 
-/** Implementation of create_iterator index_read_view callback. */
+/** Implementation of create_iterator_with_offset index_read_view callback. */
 template <bool USE_HINT>
 static int
-tree_read_view_create_iterator(struct index_read_view *base,
-			       enum iterator_type type,
-			       const char *key, uint32_t part_count,
-			       const char *pos,
-			       struct index_read_view_iterator *iterator)
+tree_read_view_create_iterator_with_offset(
```

Comment:
Made it raise an error if the CE read view iterator is created with the offset given.

Reply:
Seems this is a dead code, asserted `offset == 0` as it's done for other parameters.

The generic implementation performs a regular iterator skip instead of throwing anerror BTW.